src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -339,6 +335,10 @@ public void CB_COMMANDER_MOVE(LoginConnection conn, Packet packet)
 			var d1 = packet.GetFloat();	// ?
 			var d2 = packet.GetFloat();	// ?
 
+			// On a new character creation, this packet is sent with the index as this byte.
+			if (index == 0xFF)
+				return;
```

Comment:
I'm not sure what the purpose of this `0xFF` byte is. It is hardcoded like this in the client.

Reply:
Hm, maybe a reset of some sort.